gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/hive/HiveDataset.java

```diff
@@ -85,6 +85,8 @@
   public static final String DATASET_NAME_PATTERN_KEY = "hive.datasetNamePattern";
   public static final String DATABASE = "Database";
   public static final String TABLE = "Table";
+  public static final String DATASET_STAGING_PATH = "dataset.staging.path";
+  public static final String DATASET_PREFIX_REPLACEMENT = "hive.dataset.copy.target.table.prefixReplacement";
```

Comment:
Can you use the existing variable for this instead of inventing a new variable ?